### DISCUSSION TITLE
米游社查询变连连看了

### DISCUSSION THREAD

**Discussion by DongGua-Jun** [Arondight/Adachi-BOT#499] [General]
米哈游你坏事做尽.jpg
![image](https://user-images.githubusercontent.com/75875483/146383189-737ef698-a26a-4657-8e10-77a16d1f28ee.png)

**Comment by mark9804** [comment]
更新一下代码，虽然更新完之后会导致只能查询八个人物，但是我们现在也做不了更多了

**Reply by DongGua-Jun** [reply]
> 

一定是tx干的！
